source/mapping-rules/mapping-rule-options/embedded-documents.txt

```diff
@@ -22,21 +35,31 @@ table it is mapping from is on the one side of the foreign key relationship.
 The table containing the foreign key must also be mapped to a 
 collection in the MongoDB model.
 
-Behavior
---------
+Steps
+-----
 
-Use the :guilabel:`Embedded Documents` mapping rule to denormalize
-a foreign key relationship. With :guilabel:`Embedded Documents` you can
-nest child foreign key fields in a parent collection.
+.. procedure::
+   :style:  connected
 
-You have two main options when denormalizing your relational data 
-with the :guilabel:`Embedded Documents` option:
+   .. step:: Select a collection
 
-- You can embed the child documents into the parent and designate a 
-  :guilabel:`Prefix` and :guilabel:`Field name` to embed fields as sub documents.
+      a. Navigate to the :guilabel:`Mapping` screen.
+      #. On the :guilabel:`Schema model` pane under the 
+         :guilabel:`MongoDB` header, click on a collection.
+   
+   .. step:: Create or edit a mapping rule
+     
+      a. From the :guilabel:`Mapping` screen, click :guilabel:`+ Add` 
+         to create a new mapping rule or click the 
+         :icon-fa5:`pencil-alt` icon to edit an existing rule.
 
-- You can merge into the parent, which represents the rows as 
-  fields at the parent level.
+   .. step:: (Optional) Configure mapping rule filters
```

Comment:
While this content is OK, we've introduced an inconsistency in how we are describing mapping rule filters versus other things you can do in the rule. If we mention this, we don't we also mention including/excluding fields, adding calculated fields, changing field types, and (for embedded array/docs) setting the other advanced settings?

My suggestion (for all 3 mapping rule types) would be to structure it as follows:

1. Choose the table or collection 
2. Configure the root path (for embedded array/embedded docs only)
3. Set advanced options (varies by rule type)
4. Configure fields 
5. Save and close

In many cases we can make this a summary and link to other topics, such as mapping rule filters, calculated fields, etc.